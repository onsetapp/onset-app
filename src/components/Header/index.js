import React, { Component } from 'react'
import { StyleSheet, Text } from 'react-native'

export default class Header extends Component {
  render () {
    return (
      <Text style={styles.header}>{this.props.children}</Text>
    )
  }
}

const styles = StyleSheet.create({
  header: {
    color: '#fff',
    fontSize: 80,
    textAlign: 'center',
    fontFamily: 'movie-title-ah'
  }
})
