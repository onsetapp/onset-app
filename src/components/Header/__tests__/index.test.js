import React from 'react'
import renderer from 'react-test-renderer'
import { shallow } from 'enzyme'

import Header from '..'

describe('<Header />', () => {
  let component, text
  beforeEach(() => {
    text = 'some-text'
    component = shallow(<Header>{ text }</Header>)
  })

  it('renders the correct text', () => {
    expect(component).toBeTruthy()
  })

  it('renders correctly', () => {
    const tree = renderer.create(<Header />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
