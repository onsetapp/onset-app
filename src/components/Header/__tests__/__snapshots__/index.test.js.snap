// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> renders correctly 1`] = `
<Text
  style={
    Object {
      "color": "#fff",
      "fontFamily": "movie-title-ah",
      "fontSize": 80,
      "textAlign": "center",
    }
  }
/>
`;
